ansible/roles/ocp4-workload-ccnrd-cuttingedge/tasks/remove_workload.yml

```diff
@@ -43,6 +43,11 @@
   with_list: "{{ users }}"
   ignore_errors: yes
 
+- name: remove the clusterrole che-sa
+  shell:
```

Comment:
Can you use the k8s module for this with `absent` ?

Reply:
Can we delete clusterrole with `absent`?

Reply:
Never tried it before.

Reply:
Ok. Just tested with `absent` and it works.

Thanks for the reivew.